gen/generator.go

```diff
@@ -39,6 +39,7 @@ type Generator struct {
 	fieldNamer               FieldNamer
 	simpleBytes              bool
 	skipMemberNameUnescaping bool
+	floatFmt                 string
```

Comment:
why not byte?

Reply:
I'd made it string to keep it consistent with the incoming flag which would be string. Also, in case we use byte and pass it  as empty, the generator sets it as:

```
w := jwriter.Writer{FloatFmt: "0"}
```

I could try figuring this out further if byte seems more relevant as the type of `floatFmt`, but I guess it should be ok to keep it as string?

Reply:
You can check what value is correct on code generation stage and pass default value if cmd line parameter not passed

Reply:
So currently, since the flag package does not have support for byte/rune flags, it is a bit difficult to do this. I guess it might be okay to keep it as is, since the internal method converts the string to byte and sets the default value as well.